db/db_impl/db_impl_write.cc

```diff
@@ -1765,92 +1759,6 @@ Status DBImpl::SwitchWAL(WriteContext* write_context) {
   return status;
 }
 
-Status DBImpl::HandleWriteBufferManagerFlush(WriteContext* write_context) {
```

Comment:
What's the difference between it and Flush

Reply:
This is an internal call. It requires write context.

Reply:
Then, why we can remove it now

Reply:
It's not used. The new manager can call flush on any instance, not just current instance. We use `_write_stopped` to get around write thread.